wfc3tools/wf3rej.py

```diff
@@ -142,27 +115,27 @@ def wf3rej(input, output="", crrejtab="", scalense="", initgues="",
     Examples
     --------
     >>> from wfc3tools import wf3rej
-    >>> filename = '/path/to/some/wfc3/image.fits'
-    >>> wf3rej(filename)
+    >>> from glob import glob
+    >>> infiles = glob("*flt.fits")
+    >>> wf3rej(infiles, "output.fits", verbose=True)
 
+    >>> wf3rej("*flt.fits", "output.fits", verbose=True)
+
+    >>> wf3rej("@input.lst", "output.fits", verbose=True)
     """
 
     call_list = ["wf3rej.e"]
     return_code = None
 
     infiles, dummy = parseinput.parseinput(input)
-    if "_asn" in input:
-        raise IOError("wf3rej does not accept association tables")
-    if len(parseinput.irafglob(input)) == 0:
-        raise IOError("No valid image specified")
-    if len(parseinput.irafglob(input)) > 1:
-        raise IOError("wf3rej can only accept 1 file for"
-                      "input at a time: {0}".format(infiles))
 
     for image in infiles:
         if not os.path.exists(image):
             raise IOError("Input file not found: {0}".format(image))
 
+    # Generate a comma-separated string of the input filenames
+    input = ','.join(infiles)
+
     call_list.append(input)
 
     if output:
```

Comment:
For the lines below, raising errors instead of printing statements and returning errors will cut a few lines (not necessary just a little cleaner)

Reply:
Fixed.